staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/status_strategy_test.go

```diff
@@ -136,3 +141,110 @@ func TestPrepareForUpdate(t *testing.T) {
 		}
 	}
 }
+
+const listTypeResourceSchema = `
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: foos.test
+spec:
+  group: test
+  names:
+    kind: Foo
+    listKind: FooList
+    plural: foos
+    singular: foo
+  scope: Cluster
+  versions:
+  - name: v1
+    schema:
+      openAPIV3Schema:
+        type: object
+        properties:
+          numArray:
+            type: array
+            x-kubernetes-list-type: set
+            x-kubernetes-map-type: atomic
```

Comment:
this doesn't make sense on a property of type:array, drop this line

Reply:
Updated.